package.json

```diff
@@ -15,6 +15,7 @@
     "build:css": "cleancss -o src/assets/index.min.css src/assets/index.css",
     "prebuild": "rm -rf dist/ && mkdir dist/",
     "example:fastatic": "examples/fastatic/index.js",
+    "start:fastatic": "http-server 'examples/fastatic/' -i",
```

Comment:
I'd change this to simply `start` and mount it to the entire `examples/` directory. That way new examples can also directly be viewed. And it better matches the page not being on the domain root (just like on `voorhoede.github.io/repo-name/`). So it's easier to spot incorrect links. Finally I'd add a port number (to use in the docs) and set caching to false (`-c -1`), and maybe set it to open by default (`-o`):
`"http-server './examples/' -c-1 -o -p 3676"` (3676 is "form" in T9)